PubNubUnity/Assets/Snippets/AccessManager/AccessManagerSample.cs

```diff
@@ -0,0 +1,371 @@
+// snippet.using
+using PubnubApi;
+using PubnubApi.Unity;
+
+// snippet.end
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using UnityEngine;
+
+class AccessManagerSample
+{
+    private static Pubnub pubnub;
+
+    static void PubnubInit()
+    {
+        // snippet.pubnub_init
+        //Create configuration
+        PNConfiguration pnConfiguration = new PNConfiguration(new UserId("myUniqueUserId"))
```

Comment:
not sure Unity needs to allow user to grant token?
If Yes, Then we should mention here about secret key.
Because grant token works with secret key.